src/main/java/spotify/factories/GsonFactory.java

```diff
@@ -8,6 +8,9 @@
 import org.slf4j.LoggerFactory;
 import spotify.models.generic.AbstractPlayableObject;
 
+/**
+ * This is a helper class to make generating Gson object easier. 
```

Comment:
This description is not correct, what this class does it creates and configures and instance of `Gson` class.

Also missing tags like author and version. The functions are also not commented with javadoc.

Reply:
For this class, I just started, I will add the rest. Thanks.